src/vcUnitConversion.cpp

```diff
@@ -61,3 +62,129 @@ double vcUnitConversion_ConvertTemperature(double sourceValue, vcTemperatureUnit
 
   return celciusVal;
 }
+
+vcTimeReferenceData vcUnitConversion_ConvertTimeReference(vcTimeReferenceData sourceValue, vcTimeReference sourceReference, vcTimeReference requiredReference)
+{
+  if (sourceReference == requiredReference)
+    return sourceValue;
+
+  vcTimeReferenceData result = sourceValue;
+  result.success = true;
+
+  static double const s_weekSeconds = 60.0 * 60.0 * 24.0 * 7.0;
+  static double const s_secondsBetweenEpochs_TAI_Unix = 378691200.0;
+  static double const s_secondsBetweenEpochs_TAI_GPS = 694656000.0;
+
+  //TODO this will have to be updated every time a leap second is introduced.
+  //The next leap second date may be December 2020.
+  static const double s_leapSeconds[] = {
+    78796811.0,   // 1 - 6  - 1972
```

Comment:
ok

Reply:
Just for additional context of why, at first I had no idea these were dates due to the spacing here. I couldn't figure out the significance of `1 - 6` and `1 - 12`, and then it clicked they were the dates the second was added. I think it might be due to the extra space being _after_ the 6 instead of before it?

Reply:
Yep they are dates. I've changed the format to YYYY-MM-DD